app/controllers/publishers/organisations_controller.rb

```diff
@@ -40,4 +42,18 @@ def render_draft_saved_message
     vacancy = current_organisation.all_vacancies.find(params[:from_review])
     flash.now[:success] = t("messages.jobs.draft_saved_html", job_title: vacancy.job_title)
   end
+
+  def vacancy_statistics_form(vacancy)
+    if vacancy.id == params[:invalid_form_job_id]
+      form = Publishers::VacancyStatisticsForm.new(statistics_params)
+      form.valid? # Trigger validations to add errors to form
+      form
```

Comment:
Use `tap`. https://stackoverflow.com/questions/47890598/difference-between-kernelyield-self-yieldself-and-objecttap-in-ruby/47890832#47890832